logpyle/__init__.py

```diff
@@ -1518,13 +1526,13 @@ def __call__(self) -> Optional[float]:
         return time() - self.create_time
 
 
-class CPUTime(LogQuantity):
+class WallTime(LogQuantity):
```

Comment:
Technically, this is a compatibility break (which should offer backward compatibility), but I suppose this isn't directly used, so it might not matter.